test/unit/range/container/container_of_container_test.cpp

```diff
@@ -27,12 +27,9 @@ using container_of_container_types = ::testing::Types<std::vector<std::vector<dn
 
 TYPED_TEST_SUITE(container_of_container, container_of_container_types, );
 
-using In = std::ranges::iterator_t<seqan3::concatenated_sequences<seqan3::bitcompressed_vector<seqan3::dna4> > >;
-
-static_assert(std::readable<In>);
-
 TYPED_TEST(container_of_container, concepts)
 {
+    EXPECT_TRUE(std::readable<std::ranges::iterator_t<TypeParam>>);
```

Comment:
Probably, a stupid question, but why can you replace all the stuff ou had before in the std::ranges::iterator_t just with TypeParam?

Reply:
I don't fully understand the question, but TypeParam will be any of these types (see container_of_container_types):
* `std::vector<std::vector<dna4>>`
* `concatenated_sequences<std::vector<dna4>>`
* `concatenated_sequences<bitcompressed_vector<dna4>>`

So the concept was checked only for the third instantiation, but this concept should be true for any of the given containers.

Reply:
Ah okay, thanks for the explainantion. :)